src/CodeGenerators/GateGen/Program.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using Microsoft.Omex.Gating;
+using Microsoft.Omex.System.Data;
+
+namespace Microsoft.Omex.CodeGenerators.GateGen
+{
+	/// <summary>
+	/// Generates strongly typed classes for gates
+	/// </summary>
+	public static class Program
+	{
+		/// <summary>
+		/// Main entry point from command line.
+		/// </summary>
+		/// <param name="arguments">Command line arguments</param>
+		/// <returns>0 if successfull; 1 otherwise</returns>
+		public static int Main(string[] arguments)
+		{
+			ConsoleColor originalColor = Console.ForegroundColor;
+			const string version = "2.0.0.0";
```

Comment:
nit: As a convention I believe we use Upper case with constants

Reply:
Yeah, I didn't actually check all the style because I wanted to have a move as close to the original as possible initially. I'll check these in the next PR.